tutorials/getting-started-with-flow/05__connecting-the-form-to-the-app/content.adoc

```diff
@@ -3,17 +3,19 @@
 
 :title: Part 5 - Connecting the Form to the Application
 :author: Vaadin
-:description: Learn how to communicate between components in a Vaadin application
+:description: Learn how to communicate between components in a Vaadin Flow application
 :tags: Flow, Java
 :imagesdir: ./images
 :linkattrs:
 
-In this part of the tutorial, you'll learn how to connect the `CustomerForm` component (implemented in the previous step) to the application. This includes showing and hiding the form when a row is selected or deselected in the `Grid`, making the *Save* and *Delete* buttons work, and adding a _create new customer_ option.
+This part of the tutorial shows you how to connect the `CustomerForm` component (implemented in the previous step) to the application. 
 
-== Showing and Hiding the Form
+== Showing and Hiding the Form Data
 
-The `setCustomer(Customer)` method in the `CustomerForm` class hides the form if the specified customer is null or shows it if the customer is not null. When the `MainView` is built, the form should be invisible since there's no customer selected in the `Grid` at that point. You can hide the form by setting a `null` customer in the constructor of the `MainView` class as follows:
+The `setCustomer(Customer)` method in the `CustomerForm` class hides the form data if the specified customer is `null`, and shows it if the customer is not `null`. 
```

Comment:
```suggestion
The `setCustomer(Customer)` method in the `CustomerForm` class hides the form data if the specified customer is `null`, and shows it if the customer is defined. 
```
I think this would be easier for users to read if we are not talking about nulls and not nulls.